src/bar.scss

```diff
@@ -86,6 +96,12 @@ $block: #{$fd-namespace}-bar;
     }
   }
 
+  &__left > &__element:first-child {
+    span:nth-child(1) {
```

Comment:
this will be applied for all SPAN elements and actually never for Title component which is h1...h6 element.

Reply:
removed it